lib/dndmonster.sty

```diff
@@ -5,333 +5,395 @@
-		mod={\commandkey{mod}},
-		range={\commandkey{range}},
-		targets={\commandkey{targets}},
-		dmg={\commandkey{dmg}},
-		dmgtype={\commandkey{dmgtype}},
-		plusdmg={\commandkey{plusdmg}},
-		plusdmgtype={\commandkey{plusdmgtype}},
-		ordmg={\commandkey{ordmg}},
-		ordmgwhen={\commandkey{ordmgwhen}},
-		extra={\commandkey{extra}},
-	]
+    name=Shortbow,
+    mod=+0,
+    range=80/320,
+    targets=one target,
+    dmg=\dice{1d6},
+    dmgtype=piercing,
+    plusdmg=,
+    plusdmgtype=,
+    ordmg=,
+    ordmgwhen=,
+    extra=,
+  ]{%
+  |\monsterattack|[
+    name={\commandkey{name}},
+    type=ranged,
+    mod={\commandkey{mod}},
+    range={\commandkey{range}},
+    targets={\commandkey{targets}},
+    dmg={\commandkey{dmg}},
+    dmgtype={\commandkey{dmgtype}},
+    plusdmg={\commandkey{plusdmg}},
+    plusdmgtype={\commandkey{plusdmgtype}},
+    ordmg={\commandkey{ordmg}},
+    ordmgwhen={\commandkey{ordmgwhen}},
+    extra={\commandkey{extra}},
+  ]
 }
 
 \newkeycommand\monsterattack[
-		name=Dagger,
-		enum* type={both,melee,ranged},
-		mod=+0,
-		reach=5,
-		range=20/60,
-		targets=one target,
-		dmg=\dice{1d4},
-		dmgtype=piercing,
-		plusdmg=,
-		plusdmgtype=,
-		ordmg=,
-		ordmgwhen=,
-		extra=,
-	]{%
-	\begin{monsteraction}[\commandkey{name}]
-		\IfStrEqCase{\commandkey{type}}{%
-			{melee}{\textit{\meleeattackname:} \dnd@hit{mod}, \reachname\ \commandkey{reach} \ftname}%
-			{ranged}{\textit{\rangedattackname:} \dnd@hit{mod}, \rangename\ \commandkey{range} \ftname}%
-			{both}{\textit{\meleeorrangedattackname:} \dnd@hit{mod}, \reachname\ \commandkey{reach} \ftname or \rangename\ \commandkey{range} \ftname}%
-		}[]%
-		, \commandkey{targets}.
-		\textit{\hitname:} \dnd@damage{dmg}{dmgtype}%
-		\dnd@ifplusdmg{plusdmg}{plusdmgtype}%
-		\dnd@ifordmg{ordmg}{dmgtype}{ordmgwhen}{plusdmg}{plusdmgtype}%
-		\ifcommandkey{extra}{\commandkey{extra}}{}.
-	\end{monsteraction}
+    name=Dagger,
+    enum* type={both,melee,ranged},
+    mod=+0,
+    reach=5,
+    range=20/60,
+    targets=one target,
+    dmg=\dice{1d4},
+    dmgtype=piercing,
+    plusdmg=,
+    plusdmgtype=,
+    ordmg=,
+    ordmgwhen=,
+    extra=,
+  ]{%
+  \begin{monsteraction}[\commandkey{name}]
+    \IfStrEqCase{\commandkey{type}}{%
+      {melee}{\textit{\meleeattackname:} \dnd@hit{mod}, \reachname\ \commandkey{reach} \ftname}%
+      {ranged}{\textit{\rangedattackname:} \dnd@hit{mod}, \rangename\ \commandkey{range} \ftname}%
+      {both}{\textit{\meleeorrangedattackname:} \dnd@hit{mod}, \reachname\ \commandkey{reach} \ftname or \rangename\ \commandkey{range} \ftname}%
+    }[]%
+    , \commandkey{targets}.
+    \textit{\hitname:} \dnd@damage{dmg}{dmgtype}%
+    \dnd@ifplusdmg{plusdmg}{plusdmgtype}%
+    \dnd@ifordmg{ordmg}{dmgtype}{ordmgwhen}{plusdmg}{plusdmgtype}%
+    \ifcommandkey{extra}{\commandkey{extra}}{}.
+  \end{monsteraction}
 }
 
+\ExplSyntaxOn
+\NewDocumentCommand {\DndInnateSpellLevel} { O{At\ will} m}
+  {
+    \smallskip
+    \begin{hangingpar}
```

Comment:
Using `hangingpar` here adds a package dependency on `hang`. Currently we only depend on that in the example document.

We use description lists elsewhere, but I don't think those are a great fit for this situation. So let's promote `hang` to a package dependency and mark it in the changelog.